pages/index.js

```diff
@@ -1,7 +1,187 @@
+import Triangle from '../components/svg/Triangle';
+import Image from 'next/image';
+import Link from 'next/link';
+import Service from '../components/Service';
+import WhatWeDo from '../components/WhatWeDo';
+import Blog from '../components/Blog';
+
 export default function Home() {
 	return (
 		<main>
-			<h2>Resume</h2>
+			<div className='index--wrapper'>
+				<div className='index--first'>
+					<div className='index--slider'>
+						<div className='index--ft__wrapper'>
+							<Triangle></Triangle>
+							<h3 className='index--flavortext'>Digital Dominance</h3>
+						</div>
+						<h2>Resume</h2>
+						<p className='p--resume'>
```

Comment:
`p` isn't the block. The block should always be the file name to aid in finding the styles.